### PR TITLE
fix: Remove unused FFs

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR removes the following FFs:
- IsPriceFilter
- IsEstateSizeFilter
- IsCreatorsFilter
- IsLocationFilter
- IsRentalPeriodFilter
- IsMapViewFilters
- IsRentalPriceFilterChart
- IsHandsCategory
- isEmotesV2
- buy-crosschain

### CODE REVIEW

File: webapp/src/components/AssetBrowse/AssetBrowse.tsx [hunk 1]
```diff
@@ -185,19 +182,6 @@ const AssetBrowse = (props: Props) => {
 
   let right: ReactNode
 
-  const mapTopbar = isMapViewFiltersEnabled ? (
-    <MapTopbar
```
Comment (meelrossi) [inline webapp/src/components/AssetBrowse/AssetBrowse.tsx:189]: is this ok? if teh FF is enabled we should have the first part not the second

Comment (LautaroPetaccio) [reply webapp/src/components/AssetBrowse/AssetBrowse.tsx:189]: Great catch! I've restored it

File: webapp/src/components/AssetBrowse/AssetBrowse.tsx [hunk 2]
```diff
@@ -240,9 +219,16 @@ const AssetBrowse = (props: Props) => {
...
+          {isMap && isFullscreen ? (
+            <MapTopbar
+              showOwned={showOwnedLandOnMap}
+              onShowOwnedChange={(show: boolean) => setShowOwnedLandOnMap(show)}
+            />
+          ) : (
+            <AssetTopbar />
+          )}
           {isMap ? (
-            <MapBrowse showOwned={showOwnedLandOnMap} />
```
Comment (meelrossi) [inline webapp/src/components/AssetBrowse/AssetBrowse.tsx:245]: i think you need this too

Comment (LautaroPetaccio) [reply webapp/src/components/AssetBrowse/AssetBrowse.tsx:245]: You're right!

**Review by meelrossi — APPROVED** [review]

**Pull request merged**